creusot/src/translation.rs

```diff
@@ -100,7 +100,7 @@ pub(crate) fn after_analysis(ctx: TranslationCtx) -> Result<(), Box<dyn Error>>
                 let outputs = why3.output_filenames(());
                 let crate_name = why3.crate_name(LOCAL_CRATE);
 
-                let libname = format!("{}-{}.mlcfg", crate_name.as_str(), why3.crate_types()[0]);
+                let libname = format!("{}-{}.coma", crate_name.as_str(), why3.crate_types()[0]);
```

Comment:
This has nothing to do with this PR.

Reply:
oh that's a good one, I extracted this from another branch.